instance/instance_chg.go

```diff
@@ -250,6 +250,8 @@ func (sct *SimpleChangeTracker) TaskRemoved(subflowId int, taskId string) {
 func (sct *SimpleChangeTracker) LinkAdded(linkInst *LinkInst) {
```

Comment:
Isn't this redundant information that can be retrieved from the link definition?

Reply:
@fm-tibco This is about link in state step data.   Today it just has `change` and `status`,   we don't know the link from which task to which task.  that's why I added here.   The steps save in state server which we don't have a definition.

Reply:
We enabled it for debugger mode only.